src/stories/Form.stories.js

```diff
@@ -198,6 +210,7 @@ InputError.args = {
         accountNumber: '',
         pickFruit: '',
         pickAnotherFruit: '',
+        pickState: '',
```

Comment:
Can we also add a non-empty default value for `pickState` [here](https://github.com/Expensify/App/blob/3a09b5c06b399924ae844b4c04527d9a38f0d6ce/src/stories/Form.stories.js#L194-L200)? 

During my testing, the default value didn't seem to work properly on page load.

https://user-images.githubusercontent.com/22219519/165346917-9e3abfd6-9be2-48ba-8b5e-e5f89ef8d082.mov

Reply:
I think we have to add 1 more prop in StatePicker for this change, we need to pass defaultValue to make it work

Reply:
I have added a default value for pickState, but we also have to pass defaultValue in StatePicker component, please let me know your thought on this

Reply:
That sounds similar to what I'm doing in the Datepicker [refactor](https://github.com/Expensify/App/pull/8770/files). If that's the case I think it's fine :)

Reply:
Okay, then its fine

Reply:
Hmm the default value still doesn't load for me. Can you please check? 

https://user-images.githubusercontent.com/22219519/165588310-7d3fac89-ac44-41c7-ad83-25e153a12b8f.mov

Reply:
@luacmartins we need to pass defaultValue as a prop in StatePicker here

https://github.com/jayeshmangwani/App/blob/a8d30f4de9ac1638f45eae693710b3069aa7f4f3/src/components/StatePicker.js#L42-L44
        
  `defaultValue={props.defaultValue}`


as this value is being used in the BasePicker

https://github.com/jayeshmangwani/App/blob/a8d30f4de9ac1638f45eae693710b3069aa7f4f3/src/components/Picker/BasePicker/index.js#L14